cmd/node/config/config.toml

```diff
@@ -183,10 +183,11 @@
         MaxBatchSize = 100
         MaxOpenFiles = 10
 
+# TODO: add cacher to trie snapshot
```

Comment:
I think adding a cacher can be done in this PR.

Reply:
After a talk with Sebi, realised the cacher is no longer needed.